src/ert/plugins/plugin_manager.py

```diff
@@ -56,7 +57,9 @@ def __init__(self, plugins: Sequence[object] | None = None) -> None:
         self.add_hookspecs(ert.plugins.hook_specifications)
         if plugins is None:
             self.register(ert.plugins.hook_implementations)
-            self.load_setuptools_entrypoints(_PLUGIN_NAMESPACE)
+            with warnings.catch_warnings():
```

Comment:
I think this needs a small comment about why we are doing this.

Reply:
fixed.

Reply:
Maybe something about the user not being able to action on installed deprecated plugins could be added? Just wanted something more clearly about why and less about what. Other then that we this is good.